.github/workflows/capabilities_and_config.yaml

```diff
@@ -18,17 +19,22 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v5
         with:
-          go-version-file: 'go.mod'
+          go-version-file: "go.mod"
 
       - name: Build
         run: go build -o connector ./cmd/baton-procore
 
-      - name: Run and save output
+      - name: Run and save config output
+        run: ./connector config > config_schema.json
+
+      - name: Run and save capabilities output
```

Comment:
I think this will need the client-id client-secret for the capabilities command to work

Reply:
These have been configured so it should work?
<img width="868" height="260" alt="image" src="https://github.com/user-attachments/assets/9be8a35f-1a87-4d03-b7ae-270c74c105d0" />